google/cloud/aiplatform/jobs.py

```diff
@@ -577,11 +573,6 @@ def create(
         gca_io = gca_io_compat
         gca_machine_resources = gca_machine_resources_compat
```

Comment:
Nits: 

You can used `*_compat` i.e. `gca_machine_resources_compat` or `gca_machine_resources`(need to be re defined in the import sections) directly throughout the class now, instead of specify `gca_machine_resources = gca_machine_resources_compat` as `gca_machine_resources = gca_machine_resources_v1beta1` no longer applied

Reply:
Removed assignments in favor of `*_compat` named vars